test/uk/gov/hmrc/transitmovementsrouter/generators/TestModelGenerators.scala

```diff
@@ -123,4 +131,49 @@ trait TestModelGenerators extends BaseGenerators {
     )
   }
 
+  implicit def arbitrarySdesResponse(): Arbitrary[SdesNotificationItem] = Arbitrary {
+    for {
+      filename          <- Gen.alphaNumStr
+      correlationId     <- Gen.alphaNumStr
+      checksum          <- Gen.stringOfN(4, Gen.alphaChar)
+      checksumAlgorithm <- Gen.alphaNumStr
+      notification = SdesNotification.FileProcessed
```

Comment:
use Gen.oneOf() to test it for FileProcessed and FileFailed

Reply:
Yes you are right, but accordingly status should be mapped dynamically in the tests and have updated it like that now